source/config/SettingConfig.h

```diff
@@ -235,6 +235,19 @@ namespace emp {
       return new_ptr->value;
     }
 
+    /// Add a new setting not linked to a variable
+
+    template <typename T>
+    T & AddSetting(const std::string & name,
+                   const std::string & desc="",
+                   const char option_flag='\0')
```

Comment:
Should we add optional arg args_label (or remove it from the fun above?)

Reply:
I was just trying to keep things consistent between AddSetting and AddComboSetting. I was assuming AddComboSetting is set up this way to avoid ambiguity in which version is being called (but I haven't confirmed that)